common/shlibs

```diff
@@ -3889,3 +3889,4 @@ libjcat.so.1 libjcat-0.1.2_1
 libmatio.so.9 matio-1.5.17_2
 libvips.so.42 libvips-8.9.2_1
 libvips-cpp.so.42 libvips-8.9.2_1
+libsepol.so.1 libsepol-3.0_1
```

Comment:
This change belongs to `libsepol`. The next line (`libselinux.so.1`) belongs to this change.